src/content/contributing/release-process/_index.en.md

```diff
@@ -4,154 +4,209 @@ date: 2020-03-18T16:03:26+01:00
 
-> https://travis-ci.com/github/submariner-io/lighthouse/branches
+Since `subctl` is provided by `shipyard`'s dapper base image, if there are specific changes to `subctl` which other
+projects require, you will need to create a pre-release (for example `v0.5.0-pre0`) of the `submariner-operator` project: 
 
-For this example the build can be found [here](https://travis-ci.com/github/submariner-io/lighthouse/builds/153946391).
+1) Navigate to the [releases](https://github.com/submariner-io/submariner-operator/releases) page and create the release
+   with the new version.
 
-Verify that the build successfully completes as indicated by a green checkmark at the right. At this point the images tagged with 0.2.0-rc0 will be available on quay.io at:
+2) In the `shipyard` project, edit `package/Dockerfile.shipyard-dapper-base` and set `SUBCTL_VERSION` to the new version.
 
-> https://quay.io/repository/submariner/lighthouse-agent?tab=tags
-> https://quay.io/repository/submariner/lighthouse-coredns?tab=tags
+3) Commit the change, create a pull request and merge it.
+
+### Step 1: Create release for the `shipyard` project
 
+Navigate to the [releases](https://github.com/submariner-io/shipyard/releases) page and create the release with the new
+version. This will initiate a job to build the dapper base image. Once successfully completed, the generated image
+version (`0.5.0`) should be available on Quay here: 
 
-## Step 3: Update the Operator Version References and Create a Release
+> https://quay.io/repository/submariner/shipyard-dapper-base?tab=tags
 
-Once the other builds have finished and you have 0.2.0-rc0 release tags for the submariner and lighthouse projects, you can proceed with changes to the operator.
 
-### Change Referenced Versions
+### Step 2: Create release for the `admiral` project
 
-Edit the operator [versions](https://github.com/submariner-io/submariner-operator/edit/master/pkg/versions/versions.go) file and change the project version constants to reference the new release, "0.2.0-rc0".
+1) Pin the `shipyard` dapper base image to the new version. Edit `Dockerfile.dapper` and, on the first line, change
+   the `shipyard-dapper-base` image version from `devel` to the new version (`0.5.0`).
 
+2) Update the _go.mod_ and _go.sum_ references for `shipyard` to the new version:
 
-```bash
-cd submariner-operator
-git stash
-git remote add upstream ssh://git@github.com/submariner-io/submariner-operator
-git fetch -a -v -t upstream
-git checkout remotes/upstream/master -B update-references-to-v0.2.0-rc0
-vim pkg/versions/versions.go
-# make sure we are referencing the latest submariner code (we use it for verify-connectivity and the API)
-GO111MODULES=on go get github.com/submariner-io/submariner@v0.2.0-rc0
-git push my-repo update-references-to-v0.2.0-rc0
 ```
+make shell
+go get github.com/submariner-io/shipyard@v0.5.0
+go mod vendor
+go mod tidy
+exit
+```
+
+3) Commit the changes, create a pull request and merge it.
```

Comment:
```suggestion
3) Commit the changes, create a pull request and get it reviewed and merged.
```

Reply:
I didn't explicitly mention about getting it reviewed  as I think that's just part of the merging details plus it's problematic to require these PRs to go thru the normal review  process as that could drag out a release for days with waiting for reviewers for each PR.